src/lib/include/vector_triple.h

```diff
@@ -17,6 +17,15 @@ template <> inline float _genericload(const float *source) { return *source; }
 
 template <> inline double _genericload(const double *source) { return *source; }
 
```

Comment:
I think for maintainability we will need to move these to their own header and possibly define a lot more of them for other arch, but that is a future problem.